packages/tailwind/vitest.config.ts

```diff
@@ -4,11 +4,13 @@ export default defineConfig({
 	test: {
 		coverage: {
 			reportsDirectory: './__coverage__',
-			thresholdAutoUpdate: true,
-			lines: 100,
-			functions: 100,
-			branches: 100,
-			statements: 100,
+			thresholds: {
+				autoUpdate: true,
+				lines: 100,
+				functions: 100,
+				branches: 100,
+				statements: 100,
```

Comment:
I think it’s fine to drop these down to match the current state — we can aim to get back to full coverage later.

```suggestion
				lines: 94,
				functions: 100,
				branches: 85,
				statements: 94,
```